js/interactive-guides/utils.js

```diff
@@ -40,69 +40,94 @@ var utils = (function() {
         return resultStr;
     };
 
-    var __getNameAction = function(strAction) {
-        //console.log("AAA string ", strAction);
-        var name;  
-        if (strAction.indexOf("name=") !== -1) {
-          var index = strAction.indexOf("name=") + 5;
-          //console.log("index ", index);
-          name = strAction.substring(index);
-          //console.log("name ", name);
-          var quote = name.substring(0, 1);
-          //console.log("quote ", quote);
-          var tmpString = name.substring(1);
-          //console.log("tmpString ", tmpString);
-          var lastIndex = tmpString.indexOf(quote);
-          name = name.substring(0, lastIndex + 2);
-          //console.log("name=", name);
+    var __getTitleAction = function(strAction) {
+        var title;  
+        if (strAction.indexOf("title=") !== -1) {
+            var index = strAction.indexOf("title=") + 6;
+            //console.log("index ", index);
+            title = strAction.substring(index);
+            var nextSpace = title.indexOf(" ");
+            // get content of title
+            if (nextSpace !== -1) {
+                title = title.substring(0, nextSpace);
+            }
+            // check if contain quote
+            if (title.indexOf("\"") !== -1 ||
+                title.indexOf("'") !== -1) {
+                //var quote = title.substring(0, 1);
+                //var tmpString = title.substring(1);
+                //console.log("tmpString ", tmpString);
+                //var lastIndex = tmpString.indexOf(quote);
+                //title = title.substring(0, lastIndex + 2);
+            } else { 
+                title = "'" + title + "'";
+            }
```

Comment:
The logic should be checking for opening single/double quote and then find the corresponding ending quote. Make sure the opening quote is right after title=. It should not recognize title=abc'def'.